pyart/graph/radardisplay.py

```diff
@@ -708,7 +774,19 @@ def plot_colorbar(self, mappable=None, field=None, label=None, cax=None,
             if field is None:
                 field = self.plot_vars[-1]
             label = self._get_colorbar_label(field)
-        cb = fig.colorbar(mappable, ax=ax, cax=cax)
+
+        if orient is None:
+            orient = 'vertical'
+
+        # Find the axes locations to set colorbar
+        if cax is None:
+            ax = parse_ax(ax)
+            box = make_axes_locatable(ax)
+            if orient == 'vertical':
+                cax = box.append_axes("right", size="3%", pad=0.05)
+            if orient == 'horizontal':
+                cax = box.append_axes("bottom", size="6%", pad=0.50)
```

Comment:
Make size and pad parameters in the `plot_colorbar` function.  Not important if we revert to the plt.colorbar function.